main/.vitepress/config.mjs

```diff
@@ -437,6 +437,10 @@ export default defineConfig({
                   text: 'Contract Walkthroughs',
                   link: '/guides/orchestration/getting-started/contract-walkthroughs',
                   items: [
+                    {
```

Comment:
After updating branch and resolving conflicts, you will also have to add his to navbar